File: app/src/main/java/com/example/mateusz/coffeenotes/CoffeeType.java
package com.example.mateusz.coffeenotes;

enum CoffeeType {
  ESPRESSO("Espresso"),
  LATTE("Latte"),
  FLAT_WHITE("Flat white");

  private String coffeeType;

  CoffeeType(String coffeeType) {
    this.coffeeType = coffeeType;
  }

  @Override
  public String toString() {
    return coffeeType;
  }
}

File: app/src/main/java/com/example/mateusz/coffeenotes/CoffeeType.kt
package com.example.mateusz.coffeenotes

enum class CoffeeType constructor(private val coffeeType: String) {
    ESPRESSO("Espresso"),
    LATTE("Latte"),
    FLAT_WHITE("Flat white");

    override fun toString(): String {
        return coffeeType
    }
}
